modin/pandas/test/utils.py

```diff
@@ -900,10 +900,18 @@ def execute_callable(fn, inplace=False, md_kwargs={}, pd_kwargs={}):
                 ), "Got Modin Exception type {}, but pandas Exception type {} was expected".format(
                     type(md_e), type(pd_e)
                 )
-                if raising_exceptions:
+                if raising_exceptions and isinstance(raising_exceptions, (list, tuple)):
```

Comment:
This code branch is temporary and will be removed in https://github.com/modin-project/modin/pull/6954.

Reply:
How will this complicate further development until #6954 is merged?

Reply:
In general, it shouldn’t complicate things in any way, since this parameter is still optional.